Project.toml

```diff
@@ -5,11 +5,15 @@ version = "0.2.3"
 
 [deps]
 Distances = "b4f34e82-e78d-54a5-968a-f98e89d6e8f7"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
```

Comment:
How does one figure out what versions should be in the `[compat]`?

Reply:
Either use the latest versions (since these are tested) or the oldest non-breaking release (e.g., 0.25 (=0.25.0) for Distributions instead of 0.25.1).